src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1835,24 +1774,6 @@ can be expressed as:
 
 Once executed, this query will return a single Workflow named _Create OpenStack Instance_.
 
-=== Importing/Exporting Workflows into the Catalog
```

Comment:
I think this part must not be removed. This is the description of the use of the studio to interact with the catalog. Only the name should be updated.